prow/config.yaml

```diff
@@ -201,14 +195,14 @@ branch-protection:
             wasm:
               protect: true
         test-infra:
-          protect: true
-          required_pull_request_reviews: &tide_reviews
-            require_code_owner_reviews: false
-          restrictions: *tide_restrictions
+          required_pull_request_reviews: *tide_withcodeowners_reviews
+          restrictions: *tide_withcodeowners_restrictions
+          branches: *blocked_branches
```

Comment:
This is not correct, test-infra is not part of the istio release. Please leave `protect: true` on as before